packages/workflow-endpoints-webapp/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@kie-tools/workflow-endpoints-webapp",
```

If @ricardozanini also agree, I would suggest "sonataflow-workflow-endpoints-webapp"
```suggestion
  "name": "@kie-tools/sonataflow-workflow-endpoints-webapp",
```